README.md

```diff
@@ -320,6 +319,167 @@ override func prepareForSegue(_ segue: UIStoryboardSegue, sender sender: AnyObje
 initialVC.performSegue(StoryboardSegue.Message.Back)
 ```
 
+## NSStoryboard
+
+```
+swiftgen storyboards --template storyboards-osx-default.stencil /dir/to/search/for/storyboards
```

Comment:
The `--template` option of SwiftGen doesn't expect/need you to specify the `.stencil` extension :wink: